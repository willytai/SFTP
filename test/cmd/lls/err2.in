lls -l -z
lls -la -q
lls -lah -p
lls -a -rtyu
lls -hl -rty -a -nm -r -h -yt -l
