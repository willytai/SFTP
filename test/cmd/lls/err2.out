Illegal Option for 'lls': -z
Illegal Option for 'lls': -q
Illegal Option for 'lls': -p
Illegal Option for 'lls': -r -t -u -y
Illegal Option for 'lls': -m -n -r -t -y
